### DISCUSSION TITLE
路由redirect 只支持string 如果我需要重定向的时候不丢失query参数应该如何做

### DISCUSSION THREAD

**Discussion by 2863416607** [umijs/umi#11788] [General]
![image](https://github.com/umijs/umi/assets/44248814/bc150455-846d-457e-a863-529d735e06b1)

**Comment by 2863416607** [comment]
这种方式不支持

**Comment by fz6m** [comment]
路由表配置里不支持，自己写一个组件里面做重定向带上你想要的。
